doc/backend/norms-everkuendung-api-v1.yaml

```diff
@@ -0,0 +1,153 @@
+info:
+  title: RIS Norms API für die E-Verkündung
+  version: 1.0.0
+  description: |
+    API für die Datenübertragung von der E-Verkündung zu NeuRIS.
+
+    Über die Schnittstelle zwischen E-Verkündung und NeuRIS werden die Daten aus dem Bundesgesetzblatt I und II ausgetauscht.
+
+    ## Authentifizierung
+
+    Für die Nutzung aller Endpunkte muss ein gültiger Access Token als `Authorization: Bearer <token>` übermittelt werden. Der Token kann über einen [OAuth2 Client Credentials Grant](https://datatracker.ietf.org/doc/html/rfc6749#section-4.4) abgerufen werden. Die notwendigen Informationen (URL, Client Secret) werden vom DigitalService bereitgestellt.
+
+servers:
+  - url: /api/v1
+    description: Backend
+
+paths:
+  /verkuendungen:
+    post:
+      summary: Annahme von Verkündungen.
+      description: |
+        Die Schnittstelle erlaubt das Hochladen eines Normendokumentationspaketes bis zu einer Größe von 200mb. Das Dokumentationspaket liegt in Form eines ZIP-Archivs vor (`bgbl{1|2}_{jahr}_{nr}.zip`). Es enthält folgende Daten:
+
+        - Rechtsetzungsdokument (XML, PDF)
+        - Regelungstext (XML)
+        - soweit vorhanden: Strukturierte Anlage(n) (XML)
+        - soweit vorhanden: Nicht-strukturierte Anlagen, z.B. eingebundene Bilder, PDF-Formulare (etwa: PDF, JPG, PNG)
+        - optional: konsolidierte Fassung aus der E-Gesetzgebung (PDF)
+
+        Das Archiv wird während der Anfrage innerhalb von 120 Sekunden validiert. Eine erfolgreiche Übermittelung wird mit dem Status-Code `200 OK` bestätigt.
+
+        Wenn die Validierung fehlschlägt oder nicht innerhalb dieser Zeit abgeschlossen werden kann, wird die Anfrage mit einem Fehler beendet. Falls der Fehler aufseiten von NeuRIS liegt, sollte die Übertragung nach einem gewissen Zeitraum ernet versucht werden. Dieser Zeitraum kann dem `Retry-After`-Header der Antwort entnommen werden. Wurde die Verkündung nach 5 Versuchen nicht angenommen, muss ein manueller Austausch mit der Normendokumentation stattfinden.
+
+        ## Offene Fragen
+
+        - Die Dateiformate für nicht-strukturierte Anlagen sind noch nicht abschließend geklärt.
+        - Es ist noch offen, welchen Mehrwert die konsolidierte Fassung bringt und ob sich die Übermittelung lohnt.
+        - Art und Format der Signatur müssen noch definiert werden.
+        - Es muss entschieden werden, ob nur das ZIP-Archiv und/oder die Dateien im Archiv signiert werden.
+        - Das Verfahren zum Austausch des öffentlichen Schlüssels für die Verifizierung der Signatur muss noch definiert werden.
+        - Das Verfahren zum manuellen Übertragen der Daten, wenn die Verkündung wiederholt nicht angenommen wurde, muss noch definiert werden.
+        - Sollte die Antwort auf eine erfolgreiche Anfrage weitere Informationen enthalten oder reicht der Statuscode?
+        - Das Vorgehen und ggf. die Schnittstelle für die Übergangsphase, d.h. bevor die E-Gesetzgebung Daten aus dem LEA-Editor liefert, muss noch definiert werden.
```

Comment:
Oh now i see here this open question/topic. But if we offer the openAPI in the format of when eGG is in place (a.k.a. Rechtsetzungsdokument) we should then also deliver an openAPI with the content that we will be getting before the eGG is in place right?